GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAndroidResource.java

```diff
@@ -313,6 +313,23 @@ public synchronized final InputStream getStream() throws IOException{
         return stream;
     }
 
+    /**
```

Comment:
This brings the question whether GVRAndroidResource was ever intended to be derived from. Unless explicitly designed for that then you have to assume it is not. Let's see how it is used by Srini then we'll re-evaluate whether this is the only possible way of supporting what he needs. Having users derive from our classes can be a burden long term.